pkg/agent/sysadvisor/types/helper.go

```diff
@@ -26,13 +26,17 @@ import (
 	"github.com/kubewharf/katalyst-core/pkg/util/machine"
 )
 
-// IsNumaBinding returns true iff current container is for dedicated_cores with numa binding
-// todo: support numa exclusive
+// IsNumaBinding returns true if current container is for dedicated_cores with numa binding
 func (ci *ContainerInfo) IsNumaBinding() bool {
 	return ci.QoSLevel == consts.PodAnnotationQoSLevelDedicatedCores &&
```

Comment:
maybe it's better to use `AnnotationsIndicateNUMABinding` in `util/qos/`

Reply:
modified